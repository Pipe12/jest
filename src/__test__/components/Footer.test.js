import React from 'react';
import { mount, shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = shallow(<Footer />);

  test('Render Footer component', () => {
    expect(footer.length).toEqual(1);
  });
  test('Title render', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });

});

describe('Footer snapshot', () => {
  test('Verify Footer UI', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
