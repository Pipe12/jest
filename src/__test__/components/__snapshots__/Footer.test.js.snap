// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot Verify Footer UI 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Derechos Reservados
  </p>
</div>
`;
